roles/nova-common/defaults/main.yml

```diff
@@ -23,7 +27,7 @@ nova:
   python_libvirt_version: 1.2.2-0ubuntu2~cloud0
   qemu_kvm_version: 2.0.0+dfsg-2ubuntu1.16~cloud0
   librdb1_version: 0.80.9-0ubuntu0.14.04.2~cloud0
-  glance_endpoint: http://{{ endpoints.glance }}:9292
+  glance_endpoint: "{{ endpoints.glance.url.internal }}"
```

Comment:
That would be the minimal change to accomplish our goal. A separate discussion can happen about exposing the ucarp internal IP in the catalog as part of the internalURL (something that no other service is currently doing).